src/test/regress/sql/alter_table_set_access_method_view.sql

```diff
@@ -0,0 +1,14 @@
+CREATE SCHEMA view_acceess_method_test;
+SET search_path TO view_acceess_method_test;
+
+--create the view to test alter table set access method on it
+CREATE TABLE view_test_table (id int, val int, flag bool, kind int);
+SELECT create_distributed_table('view_acceess_method_test.view_test_table','id');
```

Comment:
Info: since you set search path to our schema, you can safely omit the schema name in this call. `SET search_path .. ` is there to make it easy to read the rest of the tests in the same file.

Reply:
Fixed you as you suggested